sdk/script/management-sdk.ts

```diff
@@ -58,18 +44,24 @@ function urlEncode(strings: string[], ...values: string[]): string {
     return result;
 }
 
-export class AccountManager {
+class AccountManager {
+    public static PERMISSIONS = {
```

Comment:
I think I'd be in favor of calling it "AppPermission" as well, unless we're not thinking of this as an "enum".